states/IN/lower_dem.yaml

```diff
@@ -0,0 +1,59 @@
+bioguide:
+contact_form:
+  steps:
+    #Example: https://indianahousedemocrats.org/members/vernon-smith/full#form_individual-member-contact
+    
+    #Using MetaVariables so the config form doesn't have to be replaced often 
+    #Using Texas Lower.yaml as a guide
+
+    #https://indianahousedemocrats.org/members
+    #Link to all the memembers just have to press thier name and then click contact to get to the web form
+
+    - visit: "https://indianahousedemocrats.org/members/vernon-smith/full#form_individual-member-contact"
+    - fill_in:
+      - name: full_name
+          selector: "input[name=full_name]"
+          value: $NAME_FULL
+          required: true
+        - name: email
+          selector: "input[name=email]"
+          value: $EMAIL
+          required: true 
+        - name: street_address
+          selector: "input[name=street_address]"
+          value: $ADDRESS_STREET
+          required: true
+        - name: city
+          selector: "input[name=city]"
+          value: $ADDRESS_CITY
+          required: true 
+        - name: zip
+          selector: "input[name=zip]"
+          value: $ADDRESS_ZIP5
+          required: true 
+        - name: phone
+          selector: "input[name=phone]"
+          value: $PHONE
+          required: true 
+
+          #Skipped the file upload section
+
+         - name: subject_line
+          selector: "input[name=subject_line]"
+          value: $SUBJECT
+          required: true 
+         - name: message
+          selector: "input[name=message]"
+          value: $MESSAGE
+          required: true
+         - name: g-recaptcha-response
+          selector: '#g-recaptcha-response'
+          value: $CAPTCHA_SOLUTION  
+    - click_on:
+        - value: "button[class='button']"
```

Comment:
Two things here - First, you want a 'selector' directive to target the element.  Second, there are three buttons on the page defined like this.  if you use "button[class='button']" it will click the first button that matches that selector, which is higher on the page and not the one you are looking for.  You'll have to include additional attributes as part of the selector to hit the right button.  Give it a try!